packages/containerd.rb

```diff
@@ -0,0 +1,159 @@
+# Adapted from Arch Linux containerd PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/containerd/trunk/PKGBUILD
+
+require 'package'
+
+class Containerd < Package
+  description 'An open and reliable container runtime'
+  homepage 'https://containerd.io/'
+  version '1.6.1'
+  compatibility 'all'
+  license 'Apache'
+  source_url 'https://github.com/containerd/containerd.git'
+  git_hashtag "v#{version}"
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_armv7l/containerd-1.6.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_armv7l/containerd-1.6.1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_x86_64/containerd-1.6.1-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: 'ba0e2edf47204f1dfb0dc009cfe9e92eaa51e269dd07333783d33e9fb6b7a73d',
+     armv7l: 'ba0e2edf47204f1dfb0dc009cfe9e92eaa51e269dd07333783d33e9fb6b7a73d',
+     x86_64: 'a16adc6e1e3bf5ab88439c5dc6c31a7d49e91e93f3b9e1e6e3f41644ec23ae44'
```

Comment:
No i686 binary?

Reply:
I didn't even bother generating one. I'm spending no effort on these hosting apps trying to get them to work on i686...

Reply:
Yeah, I'm sure Docker won't work very well on 2 gb of memory max anyway.